src/renderer/pages/Governance/ui/Governance.tsx

```diff
@@ -32,6 +32,12 @@ const DelegateShards = lazy(() =>
   import('@/widgets/DelegateModal').then((module) => ({ default: module.DelegateShards })),
 );
 
+// Lazy load unlock components
+const UnlockModal = lazy(() => import('@/widgets/UnlockModal').then((module) => ({ default: module.UnlockModal })));
+const UnlockModalShards = lazy(() =>
+  import('@/widgets/UnlockModal').then((module) => ({ default: module.UnlockModalShards })),
+);
```

Comment:
let's help with treeshaking to the bundler 

```suggestion
const UnlockModal = lazy(() => import('@/widgets/UnlockModal').then(({ UnlockModal }) => (UnlockModal)));
const UnlockModalShards = lazy(() =>
  import('@/widgets/UnlockModal').then(({ UnlockModalShards }) => (UnlockModalShards)),
);
```